Components/Navigation/navigation.html

```diff
@@ -20,5 +20,25 @@
 				<li class="nav-link"><a href="#">GitHub</a></li>
 			</ul>
 		</nav>
+
+		<!-- Navigation Component Usage -->
+		<div class="wrapper">
+			<div class="component-info-div">
+				<h2 class="text-title">Navigation Component</h2>
```

Comment:
you can add a border to navigation so that it will be easily differentiable

Reply:
i added box shadow for better distinction